docs/quickstart.rst

```diff
@@ -36,7 +36,19 @@ your instance.
 
 Bootstrap
 ---------
-First, let's create a `virtualenv <https://virtualenv.pypa.io/en/stable/installation/>`_
+
+Before we begin, you want to make sure to have Cookiecutter installed. Invenio
+leverages this tool to generate the starting boilerplate for different
+components, so it will be useful to have in general.
+
+.. code-block:: shell
+
+  # Globally install or upgrade it
+  $ sudo pip install --upgrade cookiecutter
```

Comment:
I would only show the procedure to install it as user package or even install  it inside the virtual env, otherwise it looks good to me :)

Reply:
I removed the instructions to install globally and mentioned how `cookiecutter` can alternatively be installed in the virtualenv. :+1: